### DISCUSSION TITLE
Can't run this docker on Fedora core os

### DISCUSSION THREAD

**Discussion by KokonZhang** [GioF71/mpd-alsa-docker#266] [Q&A] [answered]
Can't run this docker on Fedora core os, quit right after start, show no permission to folders

**Comment by GioF71** [comment]
Hello, can you please share your configuration? docker-compose or docker run command.
Thank you

**Comment by KokonZhang** [comment]
Hi,
Exactly follow the sample page.
It works on Archlinux and Debian while using same computer and configuration.
Thanks,
Regards,
Kokon



---- Replied Message ----
| From | Giovanni ***@***.***> |
| Date | 03/25/2023 19:00 |
| To | ***@***.***> |
| Cc | ***@***.***>***@***.***> |
| Subject | Re: [GioF71/mpd-alsa-docker] Can't run this docker on Fedora core os (Discussion #266) |

Hello, can you please share your configuration? docker-compose or docker run command.
Thank you

**Comment by GioF71** [comment, marked as answer]
Hello, assuming you are talking about the first configuration, 

```text
docker run -d \
    --name=mpd-alsa \
    --rm \
    --device /dev/snd \
    -p 6600:6600 \
    -e ALSA_OUTPUT_CREATE=yes \
    -v ${HOME}/Music:/music:ro \
    -v ${HOME}/.mpd/playlists:/playlists \
    -v ${HOME}/.mpd/db:/db \
    giof71/mpd-alsa
```

you might try without the volumes, so:

```text
docker run -d \
    --name=mpd-alsa \
    --rm \
    --device /dev/snd \
    -p 6600:6600 \
    -e ALSA_OUTPUT_CREATE=yes \
    giof71/mpd-alsa
```

If that works we can change path, permissions and/or enable user mode.
Please tell me if I selected the wrong example.

**Reply by KokonZhang** [reply]
Yes, I used this example.
I will try the without volume one and let you know. Thanks.

**Reply by KokonZhang** [reply]
It works now. Thank you so much.

**Reply by GioF71** [reply]
It is probably an issue with permissions. If the playlist and db are in your home folder and you prefer that they remain there, it might be better to enable `USER_MODE`, and maybe specify `PUID` and `PGID` if they are not 1000. You can verify running `id` from your terminal.
Otherwise, you can create some path using sudo, maybe at `/var/cache/docker-data/mpd/db` and at `/var/cache/docker-data/mpd/playlists`. Of course you can use the path you want, those that I type reflect where I like to create folders to bind to a container.

**Reply by KokonZhang** [reply]
It is OK for me so far.
Thanks

**Comment by GioF71** [comment]
Closing, this should be solved. Re-open if needed.
Thanks

**Comment by GioF71** [comment]
Probably no need to close this, as if I close it, it disappears from discussions because of the predefined filter on open.
